docs/pages/how-it-works/overview.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: How it works
+sidebar_label: Overview
+---
+
+Devpod provides the ability to provision workspaces on any infrastructure. It does so by wrapping your conventential CLI tools such as kubectl, docker, gcloud etc to deploy your development environment 
+and set up everything required to run the dev container. While creating the workspace devpod deploys an agent to the machine running the container as well as to the container itself to provide useful 
+functions such as port forwarding, log streaming and more. Doing so it provides a control plane across your development environment. DevPod can be thought of as a minimal service mesh, specifically
```

Comment:
not sure if the term `service mesh` is common knowledge outside of the cloud infra community